katalog/keycloak/sm.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: keycloak-metrics
+  labels:
+    app: keycloak
+spec:
+  selector:
+    app: keycloak
+  ports:
+    - name: metrics
+      port: 9990
+      protocol: TCP
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: keycloak
+  labels:
+    k8s-app: keycloak
+spec:
+  endpoints:
+    - interval: 10s
+      port: metrics
+      scheme: http
+      path: /metrics
+  jobLabel: k8s-app
+  namespaceSelector:
+    matchNames:
+      - keycloak
+  selector:
+    matchLabels:
+      app: keycloak
```

Comment:
There are already 2 other svcs with this label, I would change that to `keycloak-metrics` to ease the service discovery done through the Service Monitor.